plugins/in_dbus/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(src
+  in_dbus.c)
+
+pkg_check_modules(DBUS REQUIRED dbus-1)
```

Comment:
We can leave this as a required dep for now, but I suspect we will need to revisit this when upstreaming.